plat/hisilicon/hikey960/hikey960_pm.c

```diff
@@ -281,7 +257,7 @@ static void hikey960_get_sys_suspend_power_state(psci_power_state_t *req_state)
 }
 
 static const plat_psci_ops_t hikey960_psci_ops = {
-	.cpu_standby			= hikey960_pwr_domain_standby,
+	.cpu_standby			= NULL,
```

Comment:
`cpu_standby` can do a simple WFI state instead of CPU_NAP. Leaving a blank hook here is not desirable. Even if kernel request CPU NAP, this hook will just do a WFI with no side effect.

Reply:
Hi @soby-mathew, thanks for suggestion. I will change this patch for 'WFI' state only and add comment for that.